src/js/models/maps/assets/CesiumVectorData.js

```diff
@@ -603,12 +607,21 @@ define([
        * @since 2.25.0
        */
       styleBillboard: function (entity, styles) {
-        if (!this.pinBuilder) {
-          this.pinBuilder = new Cesium.PinBuilder();
-        }
-        entity.billboard.image = this.pinBuilder
-          .fromColor(styles.color, styles.markerSize)
-          .toDataURL();
+        const pin = new DOMParser()
+            .parseFromString(PIN_SVG_STRING, "image/svg+xml")
+            .querySelector("svg");
+        pin.setAttribute("width", styles.markerSize);
+        pin.setAttribute("height", styles.markerSize);
+
+        const pinPath = pin.querySelector("path");
+        pinPath.setAttribute("fill", styles.color.toCssHexString());
+        pinPath.setAttribute("fill-rule", "evenodd");
+        pinPath.setAttribute("stroke", "white");
+        pinPath.setAttribute("stroke-width", styles.markerSize);
```

Comment:
Why does stroke width == marker size?

Reply:
Doug handed this config to me so I'm not fully sure how it works. But here is a resulting svg string:

`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" width="24" height="24">\x3C!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M215.7 499.2C267 435 384 279.4 384 192C384 86 298 0 192 0S0 86 0 192c0 87.4 117 243 168.3 307.2c12.3 15.3 35.1 15.3 47.4 0zM192 128a64 64 0 1 1 0 128 64 64 0 1 1 0-128z" fill="#d1134c" fill-rule="evenodd" stroke="white" stroke-width="24"/></svg>`

You can see the outline only starts to show around width 10, and gets thicker when you increase the stroke width